.changeset/shaggy-boxes-exercise.md

```diff
@@ -0,0 +1,7 @@
+---
+"@blitzjs/next": patch
+"blitz": patch
+"@blitzjs/generator": patch
+---
+
+fixes: Blitz template
```

Comment:
- Updates `ts-log` peer dependency to `4.9.0`
- Removes `javascript` from `blitz new` menu
- Hot Fix the `Update Schema` when using blitz generator